### PR TITLE
DC-1060: update scripts to use snapshot based snapshot builder endpoints 

### PR DESCRIPTION
**Pull request by snf2ye**
https://broadworkbench.atlassian.net/browse/DC-1060
_____________
Given the recent updates to our snapshot builder endpoints ([DC-1018, DC-926, DC-866: Update main snapshot builder endpoints to reference snapshots instead of datasets by snf2ye · Pull Request #1671 · DataBiosphere/jade-data-repo](https://github.com/DataBiosphere/jade-data-repo/pull/1671) ), we need to update our scripts (profile_endpoints.py and setup_tdr_resources.py) to use the new snapshot endpoints. 

### Testing
* I've run these scripts successfully locally

### CODE REVIEW

File: tools/profile_endpoints.py [hunk 1]
```diff
@@ -19,17 +19,17 @@
...
+ - enumerateConcepts: Profiles the performance of the enumerateConcepts endpoint by sending GET requests
 with different domain IDs and search texts.
 
- - getConcepts: Profiles the performance of the getConcepts endpoint by sending GET requests with
+ - getConceptChildren: Profiles the performance of the getConceptChildren endpoint by sending GET requests with
  different concept IDs.
 """
 
 # Constants
 DATASETS = {
```
Comment (s-rubenstein) [inline tools/profile_endpoints.py]: Should this be SNAPSHOTS now?

**Review by s-rubenstein — APPROVED** [review]
LGTM

File: tools/profile_endpoints.py (hunk 1 above)
Comment (snf2ye) [reply tools/profile_endpoints.py]: Oh, good catch! Thanks - yes, I should change the argument to references snapshots as well.

File: tools/setupResourceScripts/setup_tdr_resources.py [hunk 2]
```diff
@@ -226,10 +226,9 @@ def delete_dataset_if_exists(name, clients):
     print(f"Deleted dataset {filtered_datasets[0].id}")
 
 
-def add_snapshot_builder_settings(clients, dataset_id, directory, snapshot_builder_settings_file):
-  with open(os.path.join("files", directory, snapshot_builder_settings_file)) as dataset_schema_json:
-    clients.datasets_api.update_dataset_snapshot_builder_settings(dataset_id, json.load(dataset_schema_json))
-  print(f"Snapshot builder settings were updated")
+def add_snapshot_builder_settings(clients, snapshot_id, directory, snapshot_builder_settings_file):
+  with open(os.path.join("files", directory, snapshot_builder_settings_file)) as snapshot_builder_settings_json:
+    clients.snapshots_api.update_snapshot_snapshot_builder_settings(snapshot_id, json.load(snapshot_builder_settings_json))
```
Comment (rjohanek) [inline tools/setupResourceScripts/setup_tdr_resources.py:231]: do we still want the print line?

**Review by rjohanek — APPROVED** [review]
looks good

**Pull request merged**